symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/UserService.java

```diff
@@ -698,9 +698,9 @@ public V2UserDetail update(@Nonnull Long userId, @Nonnull V2UserAttributes paylo
   public V1AuditTrailInitiatorList listAuditTrail(@Nonnull Long startTimestamp, Long endTimestamp,
       CursorPaginationAttribute pagination, Long initiatorId, String role) {
 
-    String before = pagination != null ? pagination.getBefore().toString() : null;
```

Comment:
in CursorPaginationAttribute the field are marked as non null
We should check where it is used to see where it can or cannot be null